src/sentry/grouping/enhancer/enhancement-configs/newstyle@2023-01-11.txt

```diff
@@ -30,10 +34,6 @@ family:native path:/usr/local/lib/**                                 -app
 family:native path:/usr/local/Cellar/**                              -app
 family:native package:linux-gate.so*                                 -app
 
-# well known path components for mac paths
-family:native package:**.app/Contents/**                             +app
-family:native package:/Users/**                                      +app
```

Comment:
I'm only moving these rules higher up to apply first. The snapshots basically reflect this no-op change:
<img width="653" alt="image" src="https://github.com/user-attachments/assets/89240208-d815-4afc-9f91-bce094165c5f" />